docker.md

```diff
@@ -13,38 +13,7 @@ Copy `env.dist` to the root folder and rename it to `.env`
 
 ### config.php file
 
-Copy the latest `config.php` and `config.allow_all.php` from https://github.com/cultuurnet/udb3-vagrant/tree/main/config/udb3-backend to the root
-
-In your `config.php` file, you have to change some of the hosts to work with Docker instead of Vagrant.
-
-You'll need to change the following lines to work with docker hosts:
-- url
-  - `http://host.docker.internal:8000`
-- search.v3.base_url
-  - `http://host.docker.internal:9000`
-- database.host
-  - `mysql`
-- cache.redis.host
-  - `redis`
-- amqp.host
-  - `rabbitmq`
-
-To make the SAPI3 proxy work you will need to add a `scheme` and `port` to the search config
-
-```
-'search' => [
-    'v3' => [
-        'base_url' => 'http://host.docker.internal:9000',
-        'api_key' => 'deb306a6-6f46-4c98-89ce-b03ec4fd11e2',
-        'scheme' => 'http',
-        'port' => 9000,
-    ],
-],
-```
-
-### pem files
-
-Copy `public.pem` and `public-auth0.pem` from https://github.com/cultuurnet/udb3-vagrant/tree/main/config/keys to the root
+Copy `docker-config.php` to the root folder and rename it to `config.php`
```

Comment:
Maybe the PHP docker image can do that by itself in its Dockerfile? (https://github.com/cultuurnet/udb3-backend/blob/master/docker/php/Dockerfile)

Reply:
I am afraid it cannot be done in the `Dockerfile`. That creates the container.
Only in the `docker-compose.yml` (after the dockerfile) the local filesystem is linked to the container.
Alternatively it could be done in the shellscript, so that we automatically create the `config.php` & `.env` with a `make config`

Reply:
Sounds good to me!